src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-detail-details/sw-order-detail-details.spec.js

```diff
@@ -110,6 +110,7 @@ async function createWrapper() {
                         value: 0,
                     },
                 },
+                'mt-textarea': true,
```

Comment:
Please write a proper test without using wrapper.vm internals to mock events. I could change the complete logic of the text area and your test would still be green.

Reply:
Can you please remove the mock? All meteor components are loaded in Jest by default no need to mock it with `mt-textarea: true`.